application/src/main/java/org/opentripplanner/gtfs/mapping/BikeAccessMapper.java

```diff
@@ -5,64 +5,23 @@
 import org.opentripplanner.transit.model.network.BikeAccess;
 
 /**
- * Model bike access for GTFS trips.
- * <p>
- * The GTFS bike extensions is originally discussed at: https://groups.google.com/d/msg/gtfs-changes/QqaGOuNmG7o/xyqORy-T4y0J
- * <p>
- * It proposes "route_bikes_allowed" in routes.txt and "trip_bikes_allowed" in trips.txt with the
- * following semantics:
- * <p>
- * 2: bikes allowed<br> 1: no bikes allowed<br> 0: no information (same as field omitted)<br>
- * <p>
- * The values in trips.txt override the values in routes.txt.
- * <p>
- * An alternative proposal is discussed in: https://groups.google.com/d/msg/gtfs-changes/rEiSeKNc4cs/gTTnQ_yXtPgJ
- * <p>
- * Here, the field "bikes_allowed" is used in both routes.txt and trip.txt with the following
- * semantics:
- * <p>
- * 2: no bikes allowed<br> 1: bikes allowed<br> 0: no information (same as field omitted)<br>
- * <p>
- * Here, the 0,1,2 semantics have been changed to match the convention used in the
- * "wheelchair_accessible" field in trips.txt.
- * <p>
- * A number of feeds are still using the original proposal and a number of feeds have been updated
- * to use the new proposal. For now, we support both, using "bikes_allowed" if specified and then
- * "trip_bikes_allowed".
+ * Model bike access for GTFS trips by using the bikes_allowed fields from route and trip.
  */
 class BikeAccessMapper {
 
   public static BikeAccess mapForTrip(Trip rhs) {
-    //noinspection deprecation
-    return mapValues(rhs.getBikesAllowed(), rhs.getTripBikesAllowed());
+    return mapValues(rhs.getBikesAllowed());
   }
 
   public static BikeAccess mapForRoute(Route rhs) {
-    //noinspection deprecation
-    return mapValues(rhs.getBikesAllowed(), rhs.getRouteBikesAllowed());
+    return mapValues(rhs.getBikesAllowed());
   }
```

Comment:
`bikes_allowed` only seems to be present in `trips.txt`. Should the route based handling also be removed?
https://gtfs.org/documentation/schedule/reference/#routestxt

Reply:
I would also have a knock-on effect on the `Route` type so that would also require more discussion.

Reply:
Pretty sure we don't have the bikes_allowed information in routes.txt in any of our data sets.